src/restic/node.go

```diff
@@ -493,12 +559,35 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 	case "fifo":
 	case "socket":
 	default:
-		err = errors.Errorf("invalid node type %q", node.Type)
+		return errors.Errorf("invalid node type %q", node.Type)
+	}
+
+	if err = node.fillExtendedAttributes(path); err != nil {
+		return err
 	}
 
 	return err
 }
 
+func (node *Node) fillExtendedAttributes(path string) error {
+	if node.Type == "symlink" && node.LinkTarget == "invalid" {
```

Comment:
The extended attribute library tries to follow the link. If the link does not exist, an error is raised.
Actually, node_test, TestNodeRestoreAt fails then for LinkTarget: "invalid.